x-pack/plugins/maps/public/classes/sources/es_search_source/es_search_source.tsx

```diff
@@ -537,17 +537,6 @@ export class ESSearchSource extends AbstractESSource implements IMvtVectorSource
           inspectorAdapters
         );
 
-    const unusedMetaFields = indexPattern.metaFields.filter((metaField) => {
-      return !['_id', '_index'].includes(metaField);
-    });
-    const flattenHit = (hit: Record<string, any>) => {
-      const properties = indexPattern.flattenHit(hit);
-      // remove metaFields
-      unusedMetaFields.forEach((metaField) => {
-        delete properties[metaField];
-      });
-      return properties;
-    };
```

Comment:
the shared implementation of flattenHit [ignores these types of fields by default](https://github.com/elastic/kibana/blob/8.4/src/plugins/data/common/search/tabify/tabify_docs.ts#L24-L29).